### PR TITLE
Implementing Recommended OS Project Requirements

### PR DESCRIPTION
**Pull request by Tiliavir**
## Preface
- change requested by #1 since we are no longer private ...
- This PR is based on Baloise's OS guidelines and copied from https://github.com/baloise/repository-template-java

## Following things need to be discussed
- Do we want / have to use the demo projects package structure (com.baloise.open.*) or do we stick to the existing one (ch.basler.*) - why?
- is it possible to rename LICENSE.txt to LICENSE.md for consistency with other files (breaking consistency with other projects ...)?
- please review "Branch protection rules" -  they were adjusted to enforce signed commits (as requested by CONTRIBUTING.md) and PRs are enforced at the moment (except for repo and orga admins)

### CODE REVIEW

**Comment by MarkusTiede** [comment]
@Tiliavir : good point - I‘ll add a short explanation to the guidelines explaining the recommended namespace!

File: CODEOWNERS.md [hunk 1]
```diff
@@ -0,0 +1 @@
+* @baloise/open-source
```
Comment (MarkusTiede) [inline CODEOWNERS.md]: Rather than using the group I'd recommend adding the actual project maintainer (github handles) here.

File: README.md [hunk 2]
```diff
@@ -11,4 +11,4 @@ Project to replace the old cut
 
 
 ## local setup for postgres running in docker 
- docker run --name postgres_cat -e POSTGRES_DB=cat -e POSTGRES_USER=cat -e  POSTGRES_PASSWORD=cat -d -p 5433:5432 postgres:12-alpine
\ No newline at end of file
+ docker run --name postgres_cat -e POSTGRES_DB=cat -e POSTGRES_USER=cat -e  POSTGRES_PASSWORD=cat -d -p 5433:5432 postgres:12-alpine
```
Comment (MarkusTiede) [inline README.md]: I assume that these are test data sets.

File: CODEOWNERS.md (hunk 1 above)
Comment (Tiliavir) [reply CODEOWNERS.md]: done in cea0962a011f78c31fb23e5f4a246a514604bda6

File: README.md (hunk 2 above)
Comment (Tiliavir) [reply README.md]: There are no (test/prod) data sets in the repo ... maybe we'll have to create a test data in the future ...

**Comment by MarkusTiede** [comment]
Iirc the License.txt instead of .md was due to the [maven 3rd party license plugin behavior](https://github.com/baloise/repository-template-java/blob/master/pom.xml#L102) which you imho do not make use of atm.

**Pull request merged**